docs/UserGuide.md

```diff
@@ -43,10 +49,13 @@ _More details coming soon ..._
 
 Adds a new opening to the list.
 
-Format: `add /n POSITION /c COMPANY /s STATUS /d DEADLINE`
+Format: `add p/POSITION n/NAME_OF_COMPANY c/CONTACT_NUMBER e/EMAIL s/STATUS [d/KEY_DATES]…​`
+
+* `KEY_DATES` must be in the form `EVENT@DATE`, where `EVENT` is a stage in the application process and `DATE` is the date of the stage in `YYYYMMDD` format.
+* `STATUS` **must be of either ongoing, successful or rejected**.
 
 Examples:
-* `add /n Software Engineer /c Google /s ongoing /d 2023-03-14`
+* `add p/Software Engineer n/Google c/98371204 e/chinese@google.com s/ongoing d/Interview@20230314`
```

Comment:
It might be vague since we stated in the notes that words in upper case are user inputs. The user might be confused when reading the user guide if we capitalise it in the example.